### DISCUSSION TITLE
Work on bug reports related to stats package

### DISCUSSION THREAD

**Discussion by hturner** [r-devel/r-project-sprint#25] [Ideas]
This is a list of bugs related to the stats package that would be suitable for sprint participants with a strong statistics background to work on.

| Bug ID                                                    | Level        | Status        | Summary                                     | Language |
|-----------------------------------------------------------|--------------|---------------|---------------------------------------------|----------|
| [18253](https://bugs.r-project.org/show_bug.cgi?id=18253) | Intermediate | patch welcome | family/link specific starting values in glm | R        |
| [14359](https://bugs.r-project.org/show_bug.cgi?id=14359) | Intermediate | patch welcome | enhance t.test() for paired data            | R        |

Only a couple so far - I hope to add to this list.

**Comment by MetaEntropy** [comment]
Hello,

I am interested in participating to these patches.
